metaci/plan/models.py

```diff
@@ -123,9 +129,13 @@ class Meta:
         ordering = ["name", "active", "context"]
 
     def clean(self):
-        if self.trigger != "manual" and not self.regex:
+        if self.trigger not in ["manual", "status"] and not self.regex:
             raise ValidationError(
-                "Plans with a non-manual trigger type must also specify a regex."
+                "Plans with a trigger type other than Manual or Commit Status must also specify a regex to match tags or branches."
+            )
+        if self.trigger != "status" and self.commit_status_regex:
+            raise ValidationError(
+                "Only Plans with a Commit Status trigger may specify a Commit Status Regex."
             )
```

Comment:
Looks like it's currently possible to save a plan with trigger=status but no commit_status_regex, which we probably shouldn't allow.